configuration/parser_test.go

```diff
@@ -74,18 +69,15 @@ notifications:
   - name: "val2"
   - name: "car"`
 
-func (suite *ParserSuite) TestParseOverwriteUnininitializedPoiner(c *check.C) {
+func TestParseOverwriteUnininitializedPoiner(t *testing.T) {
 	config := localConfiguration{}
 
-	os.Setenv("REGISTRY_LOG_FORMATTER", "json")
-	defer os.Unsetenv("REGISTRY_LOG_FORMATTER")
```

Comment:
Should we leave in the `defer` statements? Without these, local testing could get annoying?

Reply:
[`t.Setenv`](https://pkg.go.dev/testing#T.Setenv) is not the same as `os.Setenv`. The former cleans up after itself.